tests/test_averaging.py

```diff
@@ -277,7 +278,7 @@ def get_current_state(self):
             """
             nonlocal num_calls, super_metadata, super_tensors
             num_calls += 1
-            return super_metadata, super_tensors
+            return serializer.dumps(super_metadata), super_tensors
```

Comment:
I believe you can use self.serializer here and avoid declaring the serializer at L271?